translations/ui-orders/en.json

```diff
@@ -715,15 +697,34 @@
   "settings.closingReasons.addBtn": "Add",
   "settings.closingReasons.cancelBtn": "Cancel",
   "settings.closingReasons.editBtn": "Edit",
+  "settings.closingReasons.reason.Canceled": "Canceled",
+  "settings.closingReasons.reason.Ceased": "Ceased",
+  "settings.closingReasons.reason.Complete": "Complete",
+  "settings.closingReasons.reason.Duplication": "Duplication",
+  "settings.closingReasons.reason.Error": "Error",
+  "settings.closingReasons.reason.Lack of funds": "Lack of funds",
+  "settings.closingReasons.reason.Lack of use": "Lack of use",
+  "settings.closingReasons.reason.Licensing terms (unacceptable)": "Licensing terms (unacceptable)",
+  "settings.closingReasons.reason.Low quality": "Low quality",
+  "settings.closingReasons.reason.Merged with another title": "Merged with another title",
+  "settings.closingReasons.reason.Split into other titles": "Split into other titles",
+  "settings.closingReasons.reason.Title is out of print": "Title is out of print",
+  "settings.closingReasons.reason.Title received as a gift": "Title received as a gift",
+  "settings.closingReasons.reason.Title won't be published this year": "Title won’t be published this year",
+  "settings.closingReasons.reason.Title won't be published": "Title won’t be published",
+  "settings.closingReasons.reason.Transferred to another publisher": "Transferred to another publisher",
+  "settings.closingReasons.reason.Unpreffered format format": "Unpreferred format",
+  "settings.closingReasons.reason.Unresponsive vendor": "Unresponsive vendor",
   "settings.closingReasons.reason": "Reason",
   "settings.closingReasons.remove.error": "Reason has not been removed",
   "settings.closingReasons.remove.success": "Reason has been successfully removed",
   "settings.closingReasons.removeBtn": "Remove",
   "settings.closingReasons.save.error": "Reason has not been saved",
   "settings.closingReasons.save.success": "Reason has been successfully saved",
   "settings.closingReasons.saveBtn": "Save",
-  "settings.closingReasons.System": "System",
-  "settings.closingReasons.User": "User",
+  "settings.closingReasons.source.System": "System",
```

Comment:
the same, IMHO, it's better to use existing translations if possible

Reply:
fixed